index.html

```diff
@@ -109,63 +96,38 @@ <h1 class="title">
-        </p>
     </div>
 
     <div class="box hero is-warning">
 
         <h1 class="title">
-            For Developers
+            for developers
         </h1>
-        <p>
-            A Webxdc is <b>HTML5 content</b> wrapped into a self-contained <code>.zip</code> named <code>.xdc</code>.
-            <br><br>
-            Attached to a chat, you are already done.
-            <br><br>
-        </p>
         <div class="columns">
-            <div class="column is-two-thirds">
+            <div class="column">
                 <h2 class="subtitle">
-                    Examples
+                    documentation
                 </h2>
                 <p>
-                    You can find the examples from above at <a href="https://github.com/webxdc">github.com/webxdc</a>,
-                    improve them or <b>use them as a template</b> for your own, maybe far better ideas ❤️
-                    <br><br>
-                    If you prefer to start from scratch,
-                    you can clone the repo  <a href="https://github.com/webxdc/webxdc-dev">github.com/webxdc/webxdc-dev</a>.
+                    <a href="https://deltachat.github.io/webxdc_docs/">guide book</a>
+                    <br>
+                    <a href="https://github.com/deltachat/deltachat-core-rust/blob/master/draft/webxdc-dev-reference.md">reference</a>
                 </p>
             </div>
-            <div class="column">
+            <div class="column is-two-thirds">
                 <h2 class="subtitle">
-                    Documentation
+                    examples
                 </h2>
                 <p>
-                    <a href="https://deltachat.github.io/webxdc_docs/">Guide Book</a>
-                    <br>
-                    <a href="https://github.com/deltachat/deltachat-core-rust/blob/master/draft/webxdc-dev-reference.md">Reference</a>
+                    you can find the above examples (and more) at <a href="https://github.com/webxdc">github.com/webxdc</a> and are welcome to improve or <b>use them as a template</b> for your own, maybe far better ideas ❤️
+                    <br><br>
+                    if you prefer to start from scratch,
+                    you can clone the repo  <a href="https://github.com/webxdc/hello">github.com/webxdc/hello</a>.
                 </p>
             </div>
         </div>
     </div>
 
-    <div class="page-footer">
-        nb: Webxdc could be used by other chat messengers.
-        However, most of those are busy with introducing advertising or blockchains so we are not holding our breath right now.
-        Rest assured, when the time comes and interest rises we'll be there and open to collaboration :)
-        <br>
-        <br>
-        <br>
-        <img style="width: 3em;" alt="" src="assets/icon.png">
-        <br>
-        <a href="https://delta.chat/en/imprint">Imprint</a>
-        |
-        <a href="https://delta.chat/en/gdpr-website">Privacy Policy</a>
-    </div>
```

Comment:
this removes also the imprint/privacy footer and the repeating of the icon. maybe leave these three things, i think imprint/privacy is also needed by law.

Reply:
i reinstated imprint, logo and gdpr-website.